tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -284,16 +289,43 @@ def _gen_kernel_library(
       extra_args: Extra arguments to pass to the generator tool.
       test_tags: The tags to pass to the generated test.
       test_size: The "size" argument to pass to the test.
+      jit_i64_indexed_for_large_tensors_types: The (input|output) types for which to enable. 
+                                               JIT compilation of i64-indexed kernels for 
+                                               large inputs.
+      output_jit_i64_indexed_for_large_tensors_types: The output types for which a partial jit kernel should 
```

Comment:
nit: ...for which to enable JIT compilation of i64-indexed kernels for large inputs.

Reply:
fixed.